cupy/polynomial/polyutils.py

```diff
@@ -75,3 +75,26 @@ def as_series(alist, trim=True):
     dtype = cupy.common_type(*arrays)
     ret = [a.astype(dtype, copy=False) for a in arrays]
     return ret
+
+
+def trimcoef(c, tol=0):
```

Comment:
Please raise `ValueError` for non-1d inputs.

Reply:
Why? `as_series` already does that.

Reply:
Are you using `as_series` only for the checks of non-1d inputs? If so, please raise `ValueError` here and remove the call `as_series`.